source/how-tos/app-development/interactive/setup/enable-reverse-proxy.rst

```diff
@@ -122,12 +122,12 @@ Steps to Enable in Apache
    RHEL/CentOS 7:
      .. code-block:: sh
 
-        sudo systemctl try-restart httpd24-httpd.service httpd24-htcacheclean.service
+        sudo systemctl try-restart httpd24-httpd.service
```

Comment:
Seems like these can be tabs. Do you want to get them now or defer?

Reply:
For this particular section, I am just going to link to where we document Apache restart like other places in docs.  So these will go away and just link  to place already using tabs.

Reply:
cool